src/packages/AbstractLiveClient.ts

```diff
@@ -2,6 +2,7 @@ import { AbstractClient, noop } from "./AbstractClient";
 import { CONNECTION_STATE, SOCKET_STATES } from "../lib/constants";
 import type { DeepgramClientOptions, LiveSchema } from "../lib/types";
```

Comment:
You shouldn't need to make any changes to the Abstract client for this work. These edits should instead live inside `SpeakLiveClient.ts`